internal/process/steps/gardener_cluster_test.go

```diff
@@ -119,6 +124,74 @@ spec:
 `, k8sClient)
 }
 
+func TestCheckGardenerCluster_RunWhenReady(t *testing.T) {
+	// given
+	os := storage.NewMemoryStorage().Operations()
+	existingGC := NewGardenerCluster("runtime-id-000", "kcp-system")
+	existingGC.SetState("Ready")
+	k8sClient := fake.NewClientBuilder().WithRuntimeObjects(existingGC.ToUnstructured()).Build()
+	svc := NewCheckGardenerCluster(os, k8sClient)
```

Comment:
What is the meaning of `svc`? Is it step? So why `svc`?

Reply:
svc as a service which is tested, but a `step` would be better